src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -49,6 +49,15 @@
   .bottom-bar-area {
     position: relative;
     pointer-events: none;
+
+    .time-display {
```

Comment:
I think time-display CSS should not be in bottom bar components

Reply:
in some cases, time-display component is becoming part of the bottom-bar area.